app/views/books/_translations.html.erb

```diff
@@ -3,50 +3,57 @@ This book is available in
   <a href="/book/en">English</a>.
 </p>
 <p>
-Full translation available in
-  <a href="/book/de">Deutsch</a>,
-  <a href="/book/zh">简体中文</a>,
-  <a href="/book/zh-tw">繁體中文</a>,
-  <a href="/book/fr/v2">Français</a>,
-  <a href="/book/ja">日本語</a>,
-  <a href="/book/nl">Nederlands</a>,
-  <a href="/book/ru">Русский</a>,
-  <a href="/book/ko">한국어</a>,
-  <a href="/book/pt-br">Português (Brasil)</a> and
-  <a href="/book/cs">Čeština</a>.
+  Full translation available in
+  <table>
```

Comment:
instead of using a table wouldn't be better to use `<ul>` and `<li>` ?

Reply:
Fair enough.